src/js/redux/actions/search/searchFilterActions.js

```diff
@@ -82,6 +82,12 @@ export const updateRecipientLocations = (state) => ({
     location: state
 });
 
+// Keyword Filter
```

Comment:
Same comment here - can you move this to the top of the file to be more in order with the Sidebar filters?